### ISSUE TITLE
LLVM compilation on Linux

### ISSUE THREAD

**Issue by pierricgimmig** [google/orbit#4481]
Is LLVM supposed to always be built from source on Linux? On Ubuntu 22.04, I have `llvm-dev` and `llvm-12-dev` installed, yet the LLVM compilation starts when running `bootstrap-orbit.sh clang12_relwithdebinfo`. This is a major hindrance because of the disk size that is required to buildl LLVM, not to mention the time it takes to build on non-corporate grade machines.

**Comment by pierricgimmig** [comment]
#4309 indicates that this should be already fixed, but I'm seeing different results on my setup with the latest code.

**Comment by florian-kuebler** [comment]
With https://github.com/google/orbit/pull/4477 we went back to llvm as conan dependency (in the usual conan case).
Its unclear to me why this results in a build, but the change in conan to fix the LLVM build was merged just yesterday, so it might be just the case, that the packages are not yet deployed and conan falls back to building. @beckerhe knows more.

You might want to consider using our new system-dependency based build (e.g. `clang12_relwithdebinfo_system_deps`).

**Comment by pierricgimmig** [comment]
Ah I hadn't notice `clang12_relwithdebinfo_system_deps`, thanks for the info!

**Comment by florian-kuebler** [comment]
Yeah, that one is pretty new and actually even works without conan:
```
mkdir build && cd build
cmake -DCMAKE_BUILD_TYPE=Release                                                                                                                        \
           -DCMAKE_CXX_FLAGS="-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer" \
           ..

cmake --build .
```
